src/service/actionssdk/conversation/conversation.ts

```diff
@@ -200,6 +202,28 @@ export class Conversation<TUserStorage> {
    */
   screen: boolean
 
+  /**
+   * Set reprompts when users don't provide input to this action (no-input errors).
+   * Each reprompt represents as the {@link SimpleResponse}, but raw strings also can be specified
+   * for convenience (they're passed to the constructor of {@link SimpleResponse}).
+   *
+   * @example
+   * ```javascript
+   *
+   * app.intent('actions.intent.MAIN', conv => {
+   *   conv.noInputs = [
+   *     'Are you still there?',
+   *     new SimpleResponse('Hello?'),
```

Comment:
Maybe use a better example for `SimpleResponse`, like use different `text` and `speech` fields.